weasyl/define.py

```diff
@@ -204,21 +213,21 @@ def sql_string_series(target):
 the_fake_request = FakePyramidRequest()
 
 
-# Renders a template file and returns the result.
-
 # Caching all templates. Parsing templates is slow; we don't need to do it all
 # the time and there's plenty of memory for storing the compiled templates.
 _template_cache = {}
 
 
-def render(template_name, argv=(), cached=False):
-    if isinstance(template_name, basestring):
-        template = _template_cache.get(template_name)
-    else:
-        template = template_name
-    if template is None:
+def compile(template_name):
+    """
+    Compiles a template file and returns the result.
+    """
+    template = _template_cache.get(template_name)
+
+    if template is None or reload_templates:
```

Comment:
Considering template crap uses such a tiny amount of CPU it seems like you don't need anything more complex than this. Great! I don't know why this didn't get added earlier.